hack/managed-kyma-migration/migrate.sh

```diff
@@ -1,6 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-set -e
+set -eo pipefail
+
+function check_istio_status () {
+	local number=1
+	while [[ $number -le 100 ]] ; do
+		echo ">--> checking kyma status #$number"
+		local STATUS=$(kubectl -n kyma-system get istio default -o jsonpath='{.status.state}')
```

Comment:
Since we already know that there are clusters that will fail this check, because they have an Istio CR with a different name, should we still fail it like if the default CR wasn't ready?
I think it would be better to have a different output in case that there was already an Istio CR present and the migration requires additional steps.

Reply:
AFAIK the rename script will be run before migration script, so it should not fail in this particular case.